build.go

```diff
@@ -47,7 +54,18 @@ func Build(dependencyManager DependencyManager, clock chronos.Clock, logger scri
 		logger.Action("Completed in %s", duration.Round(time.Millisecond))
 		logger.Break()
 
-		args := `bundle exec passenger start --port ${PORT:-3000}`
+		passengerfilePath := filepath.Join(context.WorkingDir, "Passengerfile.json")
+		passengerfileConfig, err := passengerfileParser.Parse(passengerfilePath)
+		if err != nil {
+			return packit.BuildResult{}, err
+		}
+
+		port := 3000
+		if passengerfileConfig.Port != nil {
+			port = *passengerfileConfig.Port
+		}
```

Comment:
nit: I'm wondering what you think of a smaller API for this, something like:

```suggestion
		passengerfilePath := filepath.Join(context.WorkingDir, "Passengerfile.json")
		port, err := passengerfileParser.ParsePort(passengerfilePath, 3000)
		if err != nil {
			return packit.BuildResult{}, err
		}
```

Reply:
I thought about that, and in fact it's closer to what I've implemented in other buildpacks. Ultimately it comes down to whether we think we want the whole config or just subsections (like the Port) via helper methods.

I don't really have a strong opinion either way. The same logic exists in the codebase in either situation, it's more about who owns the decision of parsing the nillable port and handling the default port. I think all solutions are about equally nice and ugly.

Let me know what you prefer and I'll implement that.

Reply:
I personally prefer all of the port parsing/decision making to be delegated to a single helper if possible, and then have build just deal with the output.

Reply:
@sophiewigmore sure - works for me. At least for now. In the future if we need other fields from `Passenger.json` and we find the parser to be overloaded we can re-evaluate.

Reply:
👍